docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -231,6 +231,51 @@ docker run \
 
 The default Intel Quote Provider config is available in [Intel SGX Github repository](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/qcnl/linux/sgx_default_qcnl.conf).
 
+## Migrate from EPID Attestation to DCAP Attestation
+
+EPID attestation will be discontinued in 2025 and will no longer be available on
+any processors. All nodes using EPID attestation should migrate to DCAP
+attestation.
+
+:::info
+
+Compute node operators, please hold for further instructions. We will coordinate
+the migration in phases to ensure that during the migration the network remains
+operational.
+
+:::
+
+:::info
+
+In case you are running AESM service on Docker please see [these instructions].
+
+:::
+
+For transitioning to the DCAP attestation, follow these steps:
+1. See if your system [supports DCAP attestation]. If your hardware does not
+support DCAP attestation, you'll need to migrate your node to newer hardware.
+2. [Gracefully shutdown] your compute nodes (Sapphire and Cipher).
+3. Remove any leftover EPID attestation packages. If running on Ubuntu 22.04 run
+the following command:
+  ```bash
+    sudo apt remove libsgx-aesm-launch-plugin libsgx-aesm-epid-plugin
+  ```
+4. [Setup DCAP attestation]
+5. Restart the aesmd.service:
+  ```bash
+  sudo systemctl restart aesmd.service
+  ```
+5. [Configure the Quote Provider].
+7. Use the [attestation tool] to test if your settings are correct.
+8. Start your compute node.
+
+[these instructions]: ./set-up-trusted-execution-environment-tee.md#dcap-attestation-docker
+[supports DCAP attestation]: ../prerequisites/set-up-trusted-execution-environment-tee#aesm-service
```

Comment:
```suggestion
[supports DCAP attestation]: #aesm-service
```
I think this should work for references within same document. Same for other references. The benefit is that these will continue to work if file name ever changes.